base/values.cc

```diff
@@ -876,7 +876,36 @@ bool Value::List::empty() const {
 size_t Value::List::size() const {
   return storage_.size();
 }
+#if defined(STARBOARD)
+Value::List::iterator Value::List::begin() {
```

Comment:
Can these also be removed once we use c++20 (as well as the other customizations to this file)?

Reply:
Yes, they can. And I'm patching this part from @aee-google 's old //build config change since many net test failures can be fixed by this change as well.